extensions/objects/templates/subtitled-unicode-editor.component.ts

```diff
@@ -24,7 +24,10 @@ import { downgradeComponent } from '@angular/upgrade/static';
   templateUrl: './subtitled-unicode-editor.component.html'
 })
 export class SubtitledUnicodeEditorComponent {
-  @Input() value;
+  // These property is initialized using Angular lifecycle hooks
+  // and we need to do non-null assertion, for more information see
+  // https://github.com/oppia/oppia/wiki/Guide-on-defining-types#ts-7-1
+  @Input() value!: { _unicode: string };
```

Comment:
Why `_unicode` and not `unicode`?

Reply:
The type was declared keeping [this](https://github.com/oppia/oppia/pull/13598/files#diff-135c37ade52eb46ea97dac9e5e8b8beb4dfbb1a78599d663f201a5c4a0b1e8b0R37-R41) in view. 

I could not figure out a definite reason why it has been used with an underscore. Should it be updated without one instead?

Reply:
The link is not working for me (it doesn't show the specific location) could you tell me the name of the file you're pointing to?

Reply:
Okay, I see.